src/sparseml/pytorch/datasets/classification/imagenet.py

```
@@ -72,20 +72,26 @@ def __init__(
         train: bool = True,
         rand_trans: bool = False,
         image_size: int = 224,
+        resize_scale: float = 1.143,
+        resize_mode: str = "bilinear",
```

```suggestion
        resize_mode: Union[str, "transforms.InterpolationMode"] = "bilinear",
```

---

or we could raise an error, but I think this suggestion makes more sense